src/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/Context.cs

```diff
@@ -102,8 +102,9 @@ private bool BindSam(string target, string userName, string password)
 
             try
             {
-                if (Thread.CurrentThread.GetApartmentState() == ApartmentState.Unknown)
-                    Thread.CurrentThread.SetApartmentState(ApartmentState.MTA);
+                // jsimmons - I presumeno COM apartments in .NET Core 
+                //if (Thread.CurrentThread.GetApartmentState() == ApartmentState.Unknown)
```

Comment:
It's there.  You need to include System.Threading.Thread in project.json.  Take a look at DirectoryEntry.cs in the S.DS folder.